apps/api/src/app/inbox/utils/types.ts

```diff
@@ -29,6 +28,9 @@ export type InboxNotification = {
   secondaryAction?: Action;
   channelType: ChannelTypeEnum;
   tags?: string[];
+  redirect?: {
```

Comment:
❓  How often would someone need this property to declare a url from their backend in order to use it in their frontend? Is that more common that handling it on their frontend via the prop ?

Reply:
How often? I don't know. But the API/framework provides you more flexibility when it comes to building this URL, you might compose it from the data that is not available in your FE.

Reply:
https://github.com/novuhq/novu/pull/6227#discussion_r1703387453